cmd/benchmark.go

```diff
@@ -289,13 +289,11 @@ func (h benchmarkSourceHelper) FromUrl(s string) (fileCount uint, bytesPerFile i
 	return uint(fc), bpf, uint(nf), nil
 }
 
-var benchCmd *cobra.Command
-
 func init() {
 	raw := rawBenchmarkCmdArgs{}
 
 	// benCmd represents the bench command
-	benchCmd = &cobra.Command{
+	benchCmd := &cobra.Command{
```

Comment:
How this was working earlier, if benchCmd already some sort of global variable ?